### DISCUSSION TITLE
Turn screen off at sleep and turn on on wake up

### DISCUSSION THREAD

**Discussion by YasinAl** [mathoudebine/turing-smart-screen-python#159] [Q&A] [answered]
Hi,
Would it be possible to turn the screen off at sleep and turn on on wake up? It currently stays on the last frame when I put the PC to sleep.

**Comment by mathoudebine** [comment]
Hello @YasinAl this feature is planned for the next version! There is already an opened feature request here: #132

**Comment by mathoudebine** [comment, marked as answer]
This feature has now been developed in [Release 3.2.1](https://github.com/mathoudebine/turing-smart-screen-python/releases/tag/2.3.1)

**Reply by inv4s0r** [reply]
I'm trying to make the screen turn off at sleep too but I'm using Linux and it's not working. I tried modifying the systemd unit without success (currently it turns off after returning from sleep :( )... Any help?

Edit: Managed it by creating a second systemd unit to restart it after resume :)
